_config.yml

```diff
@@ -47,6 +47,7 @@ sched:
     histdetail: "-l -j"
     name: "-J"
     time: "-t"
+    bash: ""
```

Comment:
variable on `_config.yml`

Reply:
How about this instead:
```
bash_shebang: "#!/bin/bash"
```
that would leave us flexible enough to consider other bash locations (or `#!/usr/bin/env bash`) and still give space for adding flags